### PR TITLE
Workaround for v3.7 dockercfg secret issue

### PR DESCRIPTION
**Pull request by jcpowermac**
This PR includes the ability to download the origin v3.6.1 which
will correctly create a dockercfg secret.

### CODE REVIEW

File: ansible/group_vars/all.yml.example [hunk 1]
```diff
@@ -36,8 +36,22 @@ notify_email_replyto: noreply@example.com
 # --------------------------------
 # OpenShift client command options
 # --------------------------------
+# Path to openshift client
+oc_path: "/usr/bin/oc"
```
Comment (codificat) [inline ansible/group_vars/all.yml.example]: Well... this default can break some setups (like mine... I have ~/bin/oc symlinked to some version for easy switching). Can we make the default detect the current `oc` path?

**Review by codificat — COMMENTED** [review]
A couple of minor comments but otherwise LGTM

File: ansible/main.yml [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+      block:
+        - name: Download Origin v3.6.1 client
+          get_url:
+            url: "https://github.com/openshift/origin/releases/download/v3.6.1/openshift-origin-client-tools-v3.6.1-008f2d5-linux-64bit.tar.gz"
+            dest: /tmp/oc-client.tar
+
+        - name: Extract oc
+          unarchive:
+            src: /tmp/oc-client.tar
+            dest: /tmp/
```
Comment (codificat) [inline ansible/main.yml]: Talking about site-specific breakage, isn't `/tmp` risky? noexec configurations, strange platforms, ... just wondering

Comment (jcpowermac) [reply ansible/main.yml]: @codificat I am good to put it anywhere.  Somewhere else you think?

**Comment by jcpowermac** [comment]
@codificat added your suggestion of find `oc` binary.  Also added `no_log` as suggested by @aweiteka

**Pull request merged**